docs/Developer-guide/Building-GlusterFS.md

```diff
@@ -119,6 +119,10 @@ Please note to enable gNFS use the following flag
 
 		$ ./configure --enable-gnfs
 
+If you are looking at contributing by fixing some of the memory issues,
+use `--enable-asan` option
```

Comment:
This isn't rendered properly in markdown as there must be an empty line after this sentence and before the configure command.

Adding a little more information such as the following from the commit message will be better:


"This will build with `-fsanitize=address -fno-omit-frame-pointer` options and uses the `libasan.so` shared library, so that needs to be available."

Reply:
Done